modules/exploits/multi/misc/weblogic_deserialize_badattrval.rb

```diff
@@ -71,7 +75,7 @@ def check
     web_req = "GET /console/login/LoginForm.jsp HTTP/1.1\nHost: #{peer}\n\n"
     sock.put(web_req)
     sleep(2)
-    res = sock.get_once
+    res = sock.get
```

Comment:
Check fails over SSL if this isn't set (only headers returned). Works over clear when it is set so it doesn't break the happy path, but does this change cause any issues/lockups if it's pointed at a bad port/service?

Reply:
cc @zeroSteiner - was this fixed recently by: https://github.com/rapid7/rex-socket/pull/58 ?

Reply:
Yes that definitely looks related to what was fixed. This should be able to be rebased to pull in upstream changes and to test it.